src/invoices/InvoiceDetails/VoucherInformation/VoucherInformation.js

```diff
@@ -0,0 +1,165 @@
+      />
+    ),
+  };
+
+  return hasLoaded
+    ? (
+      <Fragment>
+        <Row>
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.status" />}
+              value={voucher.status}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.voucherNumber" />}
+              value={voucher.voucherNumber}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.voucherDate" />}
+              value={formatDate(voucher.voucherDate)}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.total" />}
+            >
+              <AmountWithCurrencyField
+                amount={voucher.amount}
+                currency={voucher.invoiceCurrency || currency}
+              />
+            </KeyValue>
+          </Col>
+        </Row>
+
+        <Row>
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.exchangeRate" />}
+              value={voucher.exchangeRate}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.disbursementNumber" />}
+              value={voucher.disbursementNumber}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.disbursementDate" />}
+              value={formatDate(voucher.disbursementDate)}
+            />
+          </Col>
+
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.disbursementAmount" />}
+            >
+              <AmountWithCurrencyField
+                amount={voucher.disbursementAmount}
+                currency={voucher.invoiceCurrency || currency}
+              />
+            </KeyValue>
+          </Col>
+        </Row>
+
+        <Row>
+          <Col xs={3}>
+            <KeyValue
+              label={<FormattedMessage id="ui-invoice.invoice.details.voucher.accountingCode" />}
+              value={voucher.accountingCode}
+            />
+          </Col>
+        </Row>
+
+        <Headline
+          margin="none"
+          size="large"
+        >
+          <FormattedMessage id="ui-invoice.invoice.details.voucher.voucherLines" />
+        </Headline>
+
+        <MultiColumnList
+          columnMapping={columnMapping}
+          columnWidths={columnWidths}
+          contentData={voucherLines}
+          formatter={resultsFormatter}
+          visibleColumns={visibleColumns}
+        />
+      </Fragment>
+    )
+    : (
```

Comment:
IMHO, it's better to use more readable `if(isLoading) { return <Loading /> }; return (<LoadedThing />)`

Reply:
Updated